spring-petclinic-vets-service/src/main/resources/application.yml

```diff
@@ -8,6 +8,13 @@ spring:
   profiles:
     active: production
 
+management:
```

Comment:
**question**: in the https://github.com/spring-petclinic/spring-petclinic-microservices-config/blob/main/application.yml#L32 shared by the all three business microservices, we already have the following line:
```
management.endpoints.web.exposure.include: "*"
```
Thus the `health` endpoint looks like to be exposed. Wasn't it?

Reply:
Yes, it's correct i forgot to check on the config repo, I will update

Reply:
Ok @MarianConstantinMarica 
For information, all Spring configuration have to be centralized into the `spring-petclinic-microservices-config` repository.